src/entity/MarketingCta.ts

```diff
@@ -0,0 +1,28 @@
+import { Column, Entity, PrimaryColumn } from 'typeorm';
+
+export type MarketingCtaVariant = 'card' | 'popover';
+
+export type MarketingCtaFlags = Partial<{
+  tagText: string;
+  tagColor: string;
+  ctaUrl: string;
+  ctaText: string;
+  description: string;
+  image: string;
+  title: string;
+}>;
+
+@Entity()
+export class MarketingCta {
+  @PrimaryColumn({ type: 'text' })
+  campaignId?: string;
+
+  @Column({ default: () => 'now()' })
+  createdAt: Date;
+
+  @Column({ type: 'text' })
+  variant?: MarketingCtaVariant;
```

Comment:
Should not be optional.
Do we go for variant or type?
(I see [type](https://dailydotdev.atlassian.net/wiki/spaces/HAN/pages/808878142/RFC-020+-+Idea+Validation+Tooling+-+marketing+posts#Arguments) here)

Reply:
Changed to variant in RFC